.circleci/config.yml

```diff
@@ -76,19 +76,15 @@ jobs:
             git config --global user.name 'FitLins CircleCI User'
             git config --global user.email 'crn.poldracklab@gmail.com'
       - run:
-          name: Install fMRIPrep derivatives of ds000003
-          command: |
-            datalad install -r ///labs/poldrack/ds003_fmriprep
-            datalad update ds003_fmriprep/
-            datalad get ds003_fmriprep/sub-0{1,2,3}/func/*_space-MNI152NLin2009cAsym_desc-*.nii.gz \
-                        ds003_fmriprep/sub-0{1,2,3}/func/*_desc-confounds_*.tsv \
-                        ds003_fmriprep/dataset_description.json \
-                        ds003_fmriprep/sub-*/*/*.json
-      - run:
-          name: Download a model for ds000003
-          command: |
-              mkdir -p ds003_models
-              cp /tmp/src/examples/models/ds000003/models/model-001_smdl.json ds003_models/
+          name: Install fMRIPrep derivatives of ds000003 and reference data
+          command: |
+            datalad install -r -s https://gin.g-node.org/shotgunosine/fitlins_tests
+            datalad update --merge fitlins_tests
+            datalad get fitlins_tests/ds003/ds003_fmriprep/sub-0{1,2,3}/func/*_space-MNI152NLin2009cAsym_desc-*.nii.gz \
+                                    fitlins_tests/ds003/ds003_fmriprep/sub-0{1,2,3}/func/*_desc-confounds_*.tsv \
+                                    fitlins_tests/ds003/ds003_fmriprep/dataset_description.json \
+                                    fitlins_tests/ds003/ds003_fmriprep/sub-*/*/*.json
+            datalad get -r fitlins_tests/ds003/nistats_smooth/ fitlins_tests/ds003/afni_smooth/ fitlins_tests/ds003/afni_blurto/ 
       - save_cache:
          key: ds003-v0-{{ .Branch }}-{{ .BuildNum }}
```

Comment:
We'll want to update all `ds003-v0` lines to `ds003-v1` to keep cache sizes down and make sure we're not depending on data from the old fetch that is no longer being kept fresh.